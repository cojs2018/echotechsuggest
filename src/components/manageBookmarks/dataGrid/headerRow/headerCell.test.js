import React from 'react';
import render from 'react-test-renderer';

import HeaderCell from "./headerCell";

describe('HeaderCell', () => {
    it('renders correctly', () => {
        const props = {
            field: 'columnId',
            headerName: 'Column Name',
            width: 25
        }

        const tree = render
            .create(<HeaderCell {...props} />)
            .toJSON();

        expect(tree).toMatchSnapshot();
    })
})