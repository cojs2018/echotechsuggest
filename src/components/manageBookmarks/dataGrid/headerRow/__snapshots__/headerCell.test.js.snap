// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderCell renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "alignContent": "center",
        "flex": 1,
        "flexDirection": "row",
        "paddingVertical": 12,
      },
      undefined,
      Object {
        "width": 25,
      },
    ]
  }
  testID="columnId"
>
  <Text
    numberOfLines={1}
    style={
      Array [
        Object {
          "color": "#000000",
          "fontFamily": "System",
          "fontWeight": "400",
          "textAlign": "left",
        },
        Array [
          Object {
            "alignItems": "center",
            "fontSize": 12,
            "fontWeight": "500",
            "height": 24,
            "lineHeight": 24,
          },
          Object {
            "color": "rgba(0, 0, 0, 0.6)",
          },
        ],
      ]
    }
  >
    Column Name
  </Text>
</View>
`;
